user_tools/src/spark_rapids_dataproc_tools/resources/profiling-conf.yaml

```diff
@@ -7,8 +7,8 @@ toolOutput:
       sparkProperties: 'Spark Properties:'
       comments: 'Comments:'
 sparkRapids:
-  jarFile: 'rapids-4-spark-tools_2.12-{}.jar'
-  repoUrl: 'https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark-tools_2.12/{}/rapids-4-spark-tools_2.12-{}.jar'
+  mvnUrl: 'https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark-tools_2.12'
+  repoUrl: '{}/{}/rapids-4-spark-tools_2.12-{}.jar'
```

Comment:
minor: could there be a generic conf shared between profiling and qualification conf so the tools jar info isn't repeated?